tests/new_api/test_device.py

```diff
@@ -44,6 +44,26 @@
         validate_observables,
     )
 
+if device_name == "lightning.kokkos":
+    from pennylane_lightning.lightning_kokkos.lightning_kokkos import (
```

Comment:
Could we put the ``kokkos`` and ``qubit`` branches together?

Reply:
Hey @multiphaseCFD, how do you suggest this implementation?